document/CoreFormat.mdk

```diff
@@ -355,12 +359,52 @@ An `issue` object describes a single issue detected by a static analysis tool.
 ### `ruleId` property { #issue-ruleId }
 
 An `issue` object shall contain a property named `ruleId` whose value is a string containing
-a stable, opaque identifier for the rule that was violated.
+the stable, opaque identifier for the rule that was violated.
 
 ~ IsoExample
 `"CA2101"`
 ~
 
+### `kind` property { #issue-kind }
+
+An `issue` object may contain a property named `kind` whose value is one of a fixed set of strings
+that specify what kind of issue this is.
```

Comment:
you don't use the language 'this is' in this document elsewhere. stylistically it jumps out. can we use something like 'that specify the category of the issue' or 'that delineate the type of issue' or some such? at your discretion